pkgs/games/sil/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv, fetchzip, ncurses, libX11, libXaw, libXt, libXext, libXmu, makeWrapper, writeScript, ... }:
+let 
+  setup = writeScript "setup" ''
+    mkdir -p "$ANGBAND_PATH"
+    # Copy all the data files into place
+    cp -ar $1/* "$ANGBAND_PATH"
+    # The copied files are not writable, make them so
+    chmod +w -R "$ANGBAND_PATH"
```

Comment:
The code should be make sure HOME is set. Otherwise it will attempt to create /.sil

Reply:
Would be symlinks also possible?

Reply:
Could do a symlink, but I'm not totally sure exactly where it ends up wanting to write. It does some odd thing with writing `.raw` files next to the data files, and I got tired of poking around in the source to try and figure out how to make it do it somewhere else.

I've fixed it to check `$HOME`. Also, if you have a better suggestion for how to setup the wrapper script I'd be interested - this feels a bit clumsy but I couldn't think of a simpler way.